HHS/Head-Start-TTADP
# [TTAHUB-2247] RTTAPA update table

## Description of change

Add a table to the RTR > RTTAPA page displaying recent communication logs where purpose = "Rttapa updates"

## How to test

Verify empty state of table. Create some communication logs. Verify that they display in the table accurately.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2247


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
